frontend/src/utils/api.js

```diff
@@ -0,0 +1,50 @@
+import { listEnvironmentGroups, useClient } from '../services';
```

Comment:
I like that we moved this to a generic location that can then be used whenever we call `listEnvironmentGroups()`

Should this be placed in `utils/api.js` though or somewhere else, such as under `modules/Shared/...`?

Reply:
I agree that `utils/api.js` is not the best place. Recaping, the frontend structure is divided into "functional" components:
- `authentication`
- `design` - UI components, contexts, hooks
- `globalErrors`
- `modules`. Each having:
  - `components`- UI components used in views
  - `views` - UI views
  - `services` - API services used in the module only
- `reauthentication`
- `services` 
     - `graphql` API services used across modules
     - `hooks` 
- `utils`

Reply:
Inside `modules/Shared` there are 3 modules used in other modules. I personally don't like this folder because shared code is sometimes defined at the top-level modules (e.g. `services`) and in this case inside modules.

In any case, I think the `useFetchGroups` constant fits better in `frontend/src/services/hooks/useFetchGroups.js`